src/main/java/com/glencoesoftware/omero/ms/image/region/ImageRegionRequestHandler.java

```diff
@@ -320,12 +320,12 @@ private void checkPlaneDef(
             Map<Long, Pixels> imagePixels = retrievePixDescription(
                     iQuery, Arrays.asList(imageRegionCtx.imageId));
             Pixels pixels = imagePixels.get(imageRegionCtx.imageId);
-            RenderingDef renderingDef =
-                    getRenderingDef(client, imageRegionCtx.imageId);
-            if (pixels != null) {
-                return getRegion(client, pixels, renderingDef);
+            if (pixels == null) {
```

Comment:
I don't think we want to change this behaviour right now. Returning null so that we get a 404 upstream is probably safest.